lib/output.js

```diff
@@ -73,7 +73,7 @@ const opportunities = (lighthouseResult, links) => {
 
   const opportunityRules = rules.filter(rule => {
     const {details} = audits[rule.id];
-    return Boolean(details) && details.type === 'opportunity' && details.overallSavingsMs > 0;
```

Comment:
This is just a no-op isn't it? I prefer to keep this rather than use a ternary.

Reply:
It seems that sometimes details is undefined, and in that case we get one of  the errors named in #120 issue.

```
TypeError: Cannot read property 'type' of undefined
```

If `details` is undefined the error will be thrown by the second check. That is the reason of modifying this return statement. 

Maybe we could follow the same approach as `loadingExperience` and set a default value for `details`

Thoughts? 

Best regards.

Reply:
In the issue, it was present when they were using `npx psi ...` which uses 4.0.1, which doesn't have the fix contained in #116, so that issue is still present. This error is fixed on HEAD and the edit here is a no-op so we don't need to touch this line to fix this error because it was previously fixed.

Reply:
Ok, I'll double check it.

Yesterday I tested all the sites present in the issue using the unpublished version with the code 
```
return Boolean(details) && details.type === 'opportunity' && details.overallSavingsMs > 0;
```
and I got the same errors detailed in the issue.

Let me check again anyway.

Many thanks @exterkamp for your close review.

Reply:
Sorry by my mistake 🤦 @exterkamp you were totally right. This change is not neccessary.

Best regards.